corehq/apps/users/views/__init__.py

```diff
@@ -741,19 +741,9 @@ def create_domain_permission_mirror(request, domain):
 @require_can_edit_or_view_web_users
 @require_GET
 def paginate_enterprise_users(request, domain):
-    limit = int(request.GET.get('limit', 10))
-    page = int(request.GET.get('page', 1))
-    skip = limit * (page - 1)
-    query = request.GET.get('query')
-
+    # Get web users
     domains = [domain] + DomainPermissionsMirror.mirror_domains(domain)
-
-    web_result = (
-        UserES().domains(domains).web_users().sort('username.exact')
-        .search_string_query(query, ["username", "last_name", "first_name"])
-        .start(skip).size(limit).run()
-    )
-    web_users = [WebUser.wrap(w) for w in web_result.hits]
+    (web_users, pagination) = _get_web_users(request, [domain])
```

Comment:
`[domain]` should be `domains`

nit: tuple brackets are unnecessary

```suggestion
    web_users, pagination = _get_web_users(request, domains)
```